include/target.mk

```diff
@@ -13,7 +13,7 @@ __target_inc=1
 DEVICE_TYPE?=router
 
 # Default packages - the really basic set
-DEFAULT_PACKAGES:=base-files libc libgcc busybox dropbear mtd uci opkg netifd fstools uclient-fetch logd
+DEFAULT_PACKAGES:=base-files libc libgcc busybox dropbear mtd uci opkg netifd fstools uclient-fetch logd ucert
```

Comment:
Maybe it would be good to know how much this change increases size of the resulting firmware image.

Reply:
@ynezz please elaborate on a proper way to do so, I remember some images are padded.

Reply:
I've just compared it:

```
before 3658658 Apr 16 12:40 build_dir/target-mips_24kc_musl/linux-ath79_generic/root.squashfs
after  3658834 Apr 16 17:33 build_dir/target-mips_24kc_musl/linux-ath79_generic/root.squashfs
diff       176 bytes
```

So that's pretty good.

Reply:
that's fairly small, isn't it?

Reply:
> that's fairly small, isn't it?

Yes it is.